test/acceptance/features/claimForCluster.feature

```diff
@@ -0,0 +1,157 @@
+            - name: type
+              value: psqlserver
+            - name: provider
+              value: aws
+          serviceEndpointDefinition:
+            - name: host
+              value: mydavphost.io
+            - name: port
+              value: "5432"
+            - name: user
+              value: davp
+            - name: password
+              value: quedicelagente
+            - name: database
+              value: davpdata
+          sla: L3
+          """
+        And On Primaza Cluster "primaza-main", RegisteredService "primaza-rsdb" state will eventually move to "Available"
+        And On Worker Cluster "primaza-worker", Resource is created
+        """
+        apiVersion: apps/v1
+        kind: Deployment
+        metadata:
+          name: stage-app
+          namespace: applications
+          labels:
+            app: stage-app
+        spec:
+          replicas: 1
+          selector:
+            matchLabels:
+              app: stage-app
+          template:
+            metadata:
+              labels:
+                app: stage-app
+            spec:
+              containers:
+              - name: bash
+                image: bash:latest
+                command: ["sleep","infinity"]
+        """
+        When On Primaza Cluster "primaza-main", Resource is created
+        """
+        apiVersion: primaza.io/v1alpha1
+        kind: ServiceClaim
+        metadata:
+          name: sc-test
+          namespace: primaza-system
+        spec:
+          serviceClassIdentity:
+          - name: type
+            value: psqlserver
+          - name: provider
+            value: aws
+          serviceEndpointDefinitionKeys:
+          - host
+          - port
+          - user
+          - password
+          - database
+          applicationClusterContext:
+            clusterEnvironmentName: primaza-worker
+            namespace: applications
+          application:
+            kind: Deployment
+            apiVersion: apps/v1
+            name: stage-app
+        """
+        Then On Primaza Cluster "primaza-main", the status of ServiceClaim "sc-test" is "Resolved"
+        And  On Worker Cluster "primaza-worker", the secret "sc-test" in namespace "applications" has the key "type" with value "psqlserver"
+        And  On Worker Cluster "primaza-worker", the service binding "sc-test" in namespace "applications" is bound
+
+    Scenario: Application Namespace does not exist
+
+        Given Primaza Cluster "primaza-main" is running
+        And Worker Cluster "primaza-worker" for "primaza-main" is running
+        And Clusters "primaza-main" and "primaza-worker" can communicate
+        And On Primaza Cluster "primaza-main", Worker "primaza-worker"'s ClusterContext secret "primaza-kw" is published
+        And On Worker Cluster "primaza-worker", application namespace "applications" exists
+        And On Worker Cluster "primaza-worker", Primaza Application Agent is deployed into namespace "applications"
```

Comment:
It seems to me this two steps should not be present, isn't it?

```suggestion
```

Reply:
The Application Agent is not getting installed in the first step.

Reply:
yes, but you're not needing here this namespace, as it is not declared in the cluster environment later

Reply:
When I remove the first step, the namespace is not getting created. So the next step fails. This is why I kept both steps.